src/Sarif/Microsoft.Json.Schema.ToDotNet.targets

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <JsonSchemaToDotNetPath>..\packages\Microsoft.Json.Schema.ToDotNet.0.6.0\tools\JsonSchemaToDotNet.exe</JsonSchemaToDotNetPath>
+    <JsonSchemaToDotNetPath>..\packages\Microsoft.Json.Schema.ToDotNet.0.7.0\tools\JsonSchemaToDotNet.exe</JsonSchemaToDotNetPath>
     <_ObjectModelOutputDirectory>$(FlavorIndependentIntermediateOutputPath)Autogenerated</_ObjectModelOutputDirectory>
```

Comment:
Version `0.7.0` of Json.Schema.ToDotNet is the version that added arbitrary dictionary support.